dist/src.78399e21.css

```diff
@@ -1,31 +1,27 @@
-html {
-  background: linear-gradient(90deg, #7e6e1c 40%, #351c80 60%);
+.card {
```

Comment:
`dist` folder needs to be removed from this repo

not enough to stop this from being merged as it already exists in the repo but something to do.

`dist` is in `.gitignore` just need to remove it from cache